doc/source/whatsnew/v0.19.0.txt

```diff
@@ -426,6 +426,7 @@ Other enhancements
      df.sort_values(by='row2', axis=1)
 
 - Added documentation to :ref:`I/O<io.dtypes>` regarding the perils of reading in columns with mixed dtypes and how to handle it (:issue:`13746`)
+- :meth:`~DataFrame.to_html` now has a ``border`` argument to control the value in the opening ``<table>`` tag. The default is the value of the ``display.html_border`` option, which defaults to 1. This also affects the notebook HTML repr. (:issue:`11563`).
```

Comment:
> This also affects the notebook HTML repr.

I understand from your initial post that this is not true?